### ISSUE TITLE
[tesla] Commands fail when car was just woken up until Eventstamp is updated

### ISSUE THREAD

**Issue by Jabe** [openhab/openhab-addons#8455]
## Expected Behavior
First ChargePort=ON (1) should open the charge port after car switches to ONLINE.  
(See screenshot of log below)

## Current Behavior
After (1) nothing happens. Car is awake but charge port doesn't open. Only after my rule detects changes to Eventstamp it'll succeed instantly (2). Eventstamp gets updated when `queryVehicleAndUpdate` is called by `[slow/fast]StateRunnable` which runs on a schedule.

## Log
![Screenshot_2020-09-11 tail -f var log openhab2 openhab log var log openhab2 events log](https://user-images.githubusercontent.com/10817/92957922-a8dcaf80-f469-11ea-857a-d2318d9b1408.png)

## Possible Solution
Make sure `isWake()` returns true after we see the car switching to ONLINE.  
-or- Simply ignore `isWake()` for commands like this. Nothing will happen if the car (Model 3) is sleeping.

## Steps to Reproduce (for Bugs)
This is the rule (mostly)

```
rule "Unlock Charge Cable"
when
    Item CAR_Blackbird_VirtUnlockChargeport received update
then
    logInfo("CAR", "Short button press")

    if (CAR_Blackbird_State.state != "online") {
        logInfo("CAR", "Need to wake car")

        var ts = CAR_Blackbird_Eventstamp.state

        var i = 0
        while (i < 12 && CAR_Blackbird_State.state != "online") {
            CAR_Blackbird_Wakeup.sendCommand(ON)
            logInfo("CAR", "Waiting for car to wake up (" + (60 - i*5) + "s remaining)")
            Thread::sleep(5000)
            i = i + 1
        }

        logInfo("CAR", "Car is awake")
        CAR_Blackbird_ChargePort.sendCommand(ON) // this won't do

        i = 0
        while (i < 12 && CAR_Blackbird_Eventstamp.state == ts) {
            logInfo("CAR", "Waiting for timestamp to change (" + (60 - i*5) + "s remaining)")
            Thread::sleep(5000)
            i = i + 1
        }

        logInfo("CAR", "Timestamp has changed")
    }

    CAR_Blackbird_ChargePort.sendCommand(ON)
end
```
## Context
Trying to hook a smart button up to unlock a third party charging cable. Press button inside. Go to garage. Car is awake and cable is unlocked. Unplug. Profit.

## Your Environment
* Version used: openHAB 2.5.8-1 (Release Build)

**Comment by lsiepel** [comment]
Been a long time since this issue was logged. If i look at [the changed comitted to this binding](https://github.com/openhab/openhab-addons/pulls?q=is%3Apr+tesla) it might allready be fixed. 

Can you verify this is fixed with the most recent snapshot verison of openHAB?

**Comment by Jabe** [comment]
Sorry, I've stopped using openHAB a long time ago.
